sdk/appcenter/src/main/java/com/microsoft/appcenter/ingestion/models/one/AppExtension.java

```diff
@@ -60,6 +70,24 @@ public void setId(String id) {
         this.id = id;
     }
 
+    /**
```

Comment:
As one collector recommends, we keep ordered JSON as in https://osgwiki.com/wiki/Common_Schema_3.0 please move name everywhere (methods, within methods for writing/reading json) between ver and locale.

Please also modify the unit tests for code coverage and regenerate equals/hashcode after moving the field to the right position.